frontend/src/scenes/funnels/funnelLogic.test.ts

```diff
@@ -111,18 +111,50 @@ describe('funnelLogic', () => {
             }
         } else if (
             url.pathname === `api/projects/${MOCK_TEAM_ID}/insights/funnel/correlation` &&
-            url.data?.funnel_correlation_type === 'events'
+            url.data?.funnel_correlation_type === 'event_with_properties'
```

Comment:
Do we not need the "events" specific endpoint any more?

Reply:
it was duplicated, same thing above

Reply:
🤦